src/components/PokemonItem.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Link } from 'react-router-dom';
+import styled from "styled-components";
+
+import Card from "./Card";
+import Icon from "./Icon";
+
+const StyledName = styled.h2`
+	text-align: center;
+	font-size: 20px;
+    justify-content: center;
+`;
+
+const PokemonItem = (props) => {
+    console.log(props.id);
+	return (
+		<Card>
+			<div
+				key={props.key}
```

Comment:
You can't use props.key here. In this case you should use props.id

More docs here: https://reactjs.org/docs/lists-and-keys.html#keys